app/services/job_manager.py

```
@@ -1782,41 +1780,19 @@ async def _process_job(self, job_id: str):
-                            "Job %s disc ID embed skipped: %s", job_id, e
-                        )
+                # Run the tool's post-convert hook. chdman uses it to embed
+                # disc-ID GAME/NAME tags into a freshly created CD/DVD CHD; the
+                # default is a no-op, so this is safe to call for every mode.
+                # The source (input_path) is still present here, even when
+                # delete-on-verify is requested (deletion happens after verify).
+                try:
+                    await registry.for_mode(job.mode.value).post_convert(
+                        input_path, job.output_path, job.mode.value,
+                    )
+                except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)